desktop_version/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ endif()
 if(${CMAKE_VERSION} VERSION_LESS "3.1.3")
     message(WARNING "Your CMake version is too old; set -std=c90 -std=c++98 yourself!")
 else()
-    set(CMAKE_C_STANDARD 90)
+    set(CMAKE_C_STANDARD 99)
```

Comment:
Is there any reason that allowing C extensions (on the next line) doesn't cut it?

Reply:
This is an FAudio thing I think. We have our own way of setting this but I'd be okay with the bump for now.

Reply:
If we're going to bump it, then the `message()` above needs to be changed too.